client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
```

Comment:
Not sure, I've seen that it's being used in [Pressable store step](https://github.com/Automattic/wp-calypso/blob/6ca5b41bedb46d17c0eae26dd17f327d04f24852/client/signup/steps/design-type-with-store/pressable-store/index.jsx#L70) for this purpose. I've tried using `page.redirect( ... )` instead, but I'm getting an error in Chrome when testing locally:

> Uncaught SecurityError: Failed to execute 'replaceState' on 'History': A history state object with URL 'https://www.bluehost.com/wordpress' cannot be created in a document with origin 'http://calypso.localhost:3000' and URL 'http://calypso.localhost:3000/start/design-type-with-store'.